Develop to Main

Miramoop opened the pull request.

Commit:
Removal of Google HTML

Commit:
Basic Quiz Questions & End Quiz Screen

Commit:
Addition of Quiz Results Page

Currently only the return to home button works

Commit:
Update Manifest.JSON & Delete Old Images

Commit:
Update QuizResults.js

CStamBizStream commented on src/App.js:
```diff
@@ -1,50 +1,135 @@
+    }
+  };
+
+  const fetchYouTubeDataAndDisplay = async () => {
```
Some of these functions could probably be broken out into a service or some utils just for general cleanup

CStamBizStream commented on public/index.html:
```diff
@@ -41,4 +41,5 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script src="https://kit.fontawesome.com/7777890cf7.js" crossorigin="anonymous"></script> 
```
Not a necessity but there is an npm package you can use for font awesome as well.

Commit:
Switch Font Awesome Icons to NPM Packages,

Commit:
Refactoring functions into services & utilities

Currently the YouTube & Spotify API calls are not functioning & results are not displayed
